### DISCUSSION TITLE
Post EMTG Publications

### DISCUSSION THREAD

**Discussion by ednasa** [nasa/EMTG#6] [Show and tell]
Hello Everyone,

Do you have any publications where EMTG was used? 
Feel free to post those publications in the comments.

Attached is a list of publications up to 2022:
[EMTG_Publications.pdf](https://github.com/user-attachments/files/18010755/EMTG_Publications.pdf)

**Comment by npn0010** [comment]
We wrote a paper on a realistic enforcement of thruster duty cycle for electric propulsion spacecraft. We implemented it in an indirect-based low thrust trajectory optimization method and we used the "realistic" duty cycle feature in EMTG for a validation and comparison of our results.

https://doi.org/10.1016/j.actaastro.2023.08.022
